tests/test_io.py

```diff
@@ -68,7 +68,7 @@ def func(fh):
 
 
 def test_coerce_to_string_io_with_path():
-    with tempfile.NamedTemporaryFile() as f:
+    with tempfile.NamedTemporaryFile(delete=False) as f:
```

Comment:
Why this change?

Reply:
The file was getting deleted as soon as it was closed and thus throwing an error so had to pass `delete=False` to avoid that.